client/src/pages/recording-view/components/plugins-pane.tsx

```diff
@@ -44,10 +45,15 @@ export const PluginsPane = () => {
   const [modalSamples, setModalSamples] = useState<Float32Array>(new Float32Array([]));
   const [modalSpectrogram, setmodalSpectrogram] = useState(null);
   const [useCloudStorage, setUseCloudStorage] = useState(true);
-  const { dataSourcesQuery } = useUserSettings();
-  const connectionInfo = dataSourcesQuery?.data[`${account}/${container}`];
+  const dataSource = getDataSource(CLIENT_TYPE_API, account, container, true).data;
```

Comment:
I think:

const {data }  = getDataSource(CLIENT_TYPE_API, account, container, true);

Is usually the preferred way of retrieving a property :)

Reply:
To get the sas token (for now)

Reply:
I would remove as we really shouldn't be using any form of sas token from the data sources or connection info going forward... Not sure why we are returning this from the API